lib/moon_web/pages/tutorials/icons/icons_page.ex

```diff
@@ -37,8 +37,8 @@ defmodule MoonWeb.Pages.IconsPage do
         <:example>
           <Icon name="files_copy" />
           <Icon name="files_copy" class="text-moon-32" />
-          <Icon name="files_copy" class="text-chi-chi-100 text-moon-48" />
-          <Icon name="files_copy" class="text-piccolo-100 text-moon-48" />
+          <Icon name="files_copy" class={"text-chi-chi-100", "text-moon-48"} />
```

Comment:
This is not required. Can be either way. This is more important in component definition.